src/routes/web.php

```diff
@@ -11,6 +11,19 @@
 |
 */
 
+use App\Http\Controllers\TodoController;
+use Illuminate\Support\Facades\Route;
+
 Route::get('/', function () {
     return view('welcome');
 });
+
+Route::get('/todo','TodoController@index')->name('todo.index');
+Route::get('/todo/create','TodoController@create')->name('todo.create');
+Route::get('/todo/{id}','TodoController@show')->name('todo.show');
+Route::get('/todo/{id}/edit','TodoController@edit')->name('todo.edit');
+Route::post('/todo','TodoController@store')->name('todo.store');
+
+Route::put('/todo/{id}','TodoController@update')->name('todo.update');
+Route::delete('/todo/{id}/delete','TodoController@delete')->name('todo.delete');
```

Comment:
[ask]
削除のルートのパスは以下が適切ですね…！
理由は考えてみてください！
```suggestion
Route::put('/todo/{id}','TodoController@update')->name('todo.update');
Route::delete('/todo/{id}','TodoController@delete')->name('todo.delete');
```

Reply:
dを指定してdeleteを送信した時点でtodoは削除がされるから/deleteは必要ないからだと考えたんですが、あっていますでしょうか？